src/wappalyze.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+/**
+ * This file allows the use of Wappalyzer without the inbuilt Zombie spider.
+ *
+ * It reads page data from a simple JSON file and outputs the found technologies as JSON.
+ */
+
+const Wappalyzer = require('../node_modules/wappalyzer/wappalyzer');
+const fs = require('fs');
+
+const wappalyzer = new Wappalyzer();
+
+const args = process.argv.slice(2);
+
+const inputFile = args[0] || '';
+
+if ( !inputFile ) {
+    process.stderr.write('No input file specified\n');
+
+    process.exit(1);
+}
+
+const document = JSON.parse(fs.readFileSync(inputFile));
+const json = JSON.parse(fs.readFileSync('node_modules/wappalyzer/apps.json'));
+
+wappalyzer.apps = json.apps;
+wappalyzer.categories = json.categories;
+
+wappalyzer.driver.log = (message, source, type) => {
```

Comment:
Would it not be a good idea to have some way of accessing these logs?

Reply:
Why're you so lazy, you're the wurst

![image](https://user-images.githubusercontent.com/1678803/32783311-2f108ab8-c943-11e7-9e86-aa6135a55be8.png)